docs/fargate.md

```diff
@@ -640,7 +665,70 @@ export oidc_provider=$(aws eks describe-cluster --name $CLUSTER --region $AWS_RE
 export service_account=$HELM_INSTALLATION_NAME-sumologic-otelcol-logs-collector
 ```
 
-Set the following trust relationship
+Create a cloudwatch policy
+
+```bash
+cat >cloudwatch-policy.json <<EOF
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "VisualEditor0",
+            "Effect": "Allow",
+            "Action": [
+                "logs:GetLogEvents",
+                "logs:ListTagsForResource"
+            ],
+            "Resource": [
+                "arn:aws:logs:*:*:log-group:*:log-stream:*",
+                "arn:aws:logs:*:*:destination:*"
+            ]
+        },
+        {
+            "Sid": "VisualEditor1",
+            "Effect": "Allow",
+            "Action": [
+                "logs:DescribeQueries",
+                "logs:DescribeExportTasks",
+                "logs:GetLogRecord",
+                "logs:GetQueryResults",
+                "logs:StopQuery",
+                "logs:TestMetricFilter",
+                "logs:DescribeQueryDefinitions",
+                "logs:DescribeResourcePolicies",
+                "logs:GetLogDelivery",
+                "logs:DescribeDestinations",
+                "logs:ListLogDeliveries"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Sid": "VisualEditor2",
+            "Effect": "Allow",
+            "Action": [
+                "logs:ListTagsLogGroup",
+                "logs:GetDataProtectionPolicy",
+                "s3:GetObject",
+                "logs:DescribeLogGroups",
+                "logs:DescribeLogStreams",
+                "logs:DescribeSubscriptionFilters",
+                "logs:StartQuery",
+                "logs:Unmask",
+                "logs:DescribeMetricFilters",
+                "logs:FilterLogEvents",
+                "logs:GetLogGroupFields",
+                "logs:ListTagsForResource"
+            ],
```

Comment:
Now that I think of it, do we need all of these permissions?

Reply:
I tested it with this policy, maybe I could reduce the permissions

Reply:
Yes, I originally also tested with this policy, but maybe we should limit this.
Unfortunately, I don't see any info on that in the `awscloudwatch` receiver's docs ☹️  https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/v0.77.0/receiver/awscloudwatchreceiver

Reply:
The manual test for this seemed to be failing, i.e the logs weren't being collected after  I removed "logs:ListTagsForResource" for example. I can run more tests and raise a future PR if this can be reduced and merge these tested permissions for now, WDYT?